x-pack/plugins/cases/public/components/connectors/resilient/use_get_incident_types.test.tsx

```diff
@@ -40,7 +39,9 @@ describe.skip('useGetIncidentTypes', () => {
       { wrapper: appMockRender.AppWrapper }
     );
 
-    await waitForNextUpdate();
+    await waitFor(() => {
+      expect(spy).toHaveBeenCalled();
+    });
```

Comment:
```suggestion
    await waitFor(() => {
      expect(spy).toHaveBeenCalledWith({
        http,
        signal: expect.anything(),
        connectorId: connector.id,
      });
    });
```

and remove line 46 and below?

Reply:
I do this because if we have the `toHaveBeenCalledWith` inside the `waitFor`, we do not know if the `expect` failed because the function did not get called with the correct arguments or did not call at all. The logs will not show anything in the CI. This way, in the case of a flaky test, the logs will show the expected args.